README.md

```diff
@@ -1 +1,158 @@
+  - [🛠 Built With](#built-with)
+  -  [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+   - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+
+
+**[Awesome-Books]** is a simple website that displays a list of books and allows you to add and remove books from that list.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+- HTML.
+- CSS.
+- Javascript.
+  
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The following are the key features of the application.
+
+- **[Add and Remove your favourite books]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] ([https://devkaranj.github.io/AwesomeBooks/](https://devkaranj.github.io/AwesomeBooks-ES6/))
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+- vscode.
+- git.
+- set up an index.html document.
+
+### Setup
+Clone this repository to your desired folder:
+
+Navigate to the  directory where you  want to clone this project in your computer.
+Clone this repository to your desired folder:git clone (https://github.com/DevKaranJ/AwesomeBooks.git)
+
+### Install
+Install this project with this command:
+`$ npm install`
+
+
+### Usage
+
+To run the project, execute the following command by opening the vscode with the command `code .`
+
+### Run tests
+
+To run tests, run the following command:
+
+Run it in your local server
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Karan Joshi**
+
+- GitHub: [@githubhandle](https://github.com/DevKaranJ)
+- Twitter: [@twitterhandle](https://twitter.com/RX0069)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dev-karan-j/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- Setting Up A Professional Github Repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Kindly fix the broken link, right now your issues links give a 404 error page because of the wrong path. Please add the correct path and make sure that it is opening your repo's issue page.